paper-menu.html

```diff
@@ -0,0 +1,330 @@
+      color: var(--disabled-text-color);
+    }
+
+    :host > ::content > .core-selected {
+      position: relative;
+    }
+
+    :host > ::content > .core-selected::after {
+      mixin(--fit);
+
+      background: currentColor;
+      /* FIXME move to paper-styles for next widget */
+      opacity: 0.12;
+      content: '';
+
+      mixin(--paper-menu-selected-item);
+    }
+
+    :host > ::content > .core-selected[colored]::after {
+      opacity: 0.26;
+
+      mixin(--paper-menu-colored-selected-item);
+    }
+
+  </style>
+
+  <template>
+
+    <content></content>
+
+  </template>
+
+</dom-module>
+
+<script>
+
+(function() {
+
+  // FIXME menu control should be refactored to a more general element
+
+  Polymer({
+
+    is: 'paper-menu',
+
+    enableCustomStyleProperties: true,
+
+    mixins: [
+      Polymer.Core.Selectable
+    ],
+
+    properties: {
+
+      /**
+       * Returns the currently focused item.
+       *
+       * @attribute focusedItem
+       * @type Object
+       */
+      focusedItem: {
+        observer: 'focusedItemChanged',
+        readOnly: true,
+        type: Object
+      },
+
+      /**
+       * The attribute to use on menu items to look up the item title. Typing the first
+       * letter of an item when the menu is open focuses that item. If unset, `textContent`
+       * will be used.
+       *
+       * @attribute attrForItemTitle
+       * @type String
+       */
+      attrForItemTitle: {
+        type: String
+      },
+
+      /***********************************************************************/
+      /* Polymer.Core.Selectable                                             */
+      /***********************************************************************/
+
+      /**
+       * If you want to use the attribute value of an element for `selected` instead of the index,
+       * set this to the name of the attribute.
+       *
+       * @attribute attrForSelected
+       * @type String
+       */
+      attrForSelected: {
+        type: String
+      },
+
+      /**
+       * If true, multiple selections are allowed.
+       *
+       * @attribute multi
+       * @type Boolean
+       * @default false
+       */
+      multi: {
+        observer: 'multiChanged',
+        type: Boolean,
+        value: false
+      },
+
+      /**
+       * Gets or sets the selected element. The default is to use the index of the item.  In
+       * multi-selection this should be an array of values.
+       *
+       * @attribute selected
+       * @type String|Array
+       */
+      selected: {
+        notify: true,
+        observer: 'selectedChanged',
+        type: String
+      },
+
+      /**
+       * Returns the currently selected item. In multi-selection this returns an array of
+       * selected items.
+       *
+       * @attribute selectedItem
+       * @type Object|Array
+       */
+      selectedItem: {
+        notify: true,
+        observer: 'selectedItemChanged',
+        readOnly: true,
```

Comment:
Since this is `readOnly`, and `_setSelectedItem` is never called, can it ever actually have a value?